### ISSUE TITLE
Separate Tap Parser

### ISSUE THREAD

**Issue by mikkoh** [Jam3/tap-browser-el#1]
Use a separate parser for parsing tap output. This should clean up the code a lot more.

**Comment by vvo** [comment]
@MikkoH Yes I thought exactly the same, it would reduce A LOT the codebase, go for https://github.com/substack/tap-parser
